duet/sys/conservative.g

```diff
@@ -0,0 +1,12 @@
+; Axis and motor configuration
+M201 X1500 Y1500 Z10 E500      ; Accelerations (mm/s^2)
+M203 X12000 Y12000 Z450 E1800   ; Maximum speeds (mm/min)
+M566 X500 Y500 Z15 E10        ; Maximum jerk speeds mm/minute
+
+; Heater model
+M307 H0 A270.7 C90.4 D6.7 B0 S1.0          ; Default Bed Heater Parameters, before tuning
+M307 H1 A508.1 C249.0 D3.8 S1.00 V24.2 B0  ; Default Tool Heater Parameters, before tuning
+
+M558 H10 F50 T3000	; Z probe
+                    ; Dive height (H8). Speeds 
+                    ; (Probe - F50, travel - T3000).
```

Comment:
I wonder if maybe this would be more clear if we commented it more like:
```
; H10 - dive height
; F50 - probing speed
; T3000 - travel speed between probe points
```
Not just here but in general on these m-codes that are not intuitive.  Even for things I've gotten familiar with, seems like I end up needing to hit the docs.

Reply:
Sure, Explaining this in the macro too would be good too.

During commissioning is a time where things go wrong more than others naturally. The other parameters are about slowing down probing especially homing - where the printer no longer knows where Z0 is any more. 
If whatever probe has been hooked up fails, then the hot-end is hitting the bed.

The H10 dive height: A bigger dive height simply where the print head resets to after a probe, or between probes.
The typical calibration is home, followed by a lead screw levelling.  A situation can occur where the bed is offset by more than the dive height on any corner. 
So the bed successfully homes (say in the centre), moves up to the dive height then when it moves to the bad leveling point to probe there. 
Your hardened steel nozzle gouges while moving in XY to the probe point (and relatively moving down into the print bed) and can allegedly gouge line through your brand new print surface.
A small dive height is bad during commissioning - don't ask why I believe this...